policy/modules/system/systemd.if

```diff
@@ -286,6 +295,128 @@ template(`systemd_user_app_status',`
 	allow $2 $1_systemd_t:process sigchld;
 ')
 
+########################################
+## <summary>
+##	Read the process state (/proc/pid) of
+##	the specified systemd user instance.
+## </summary>
+## <param name="prefix">
+##	<summary>
+##	Prefix for the user domain.
+##	</summary>
+## </param>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
+##	</summary>
+## </param>
+#
+template(`systemd_spec_user_read_state',`
```

Comment:
systemd_read_user_session_state()

Reply:
Similar comment for the following interfaces.

Reply:
Also will take further suggestion, as I'm not firm on "user_session".

Reply:
>Also will take further suggestion, as I'm not firm on "user_session".

Systemd's documentation calls the `systemd --user` process the "user manager", so maybe `systemd_read_user_manager_state()` would be better.

Reply:
Ok, stick with systemd's terminology.

Reply:
Yes session is misleading.  Any existing references to systemd "session" might have come from me, sry.
There is only one systemd --user instance no matter how many sessions you might have.